jvm-runtime/plugin/common/service.go

```diff
@@ -315,33 +318,33 @@ func (s *Service) handleState(ctx context.Context, state *hotreloadpb.SchemaStat
 	}), nil
 }
 
-func (s *Service) launchQuarkusProcessAsync(ctx context.Context, devModeBuild string, projectConfig projectconfig.Config, buildCtx buildContext, stdout *errorDetector) {
-	go func() {
```

Comment:
moved the `go` keyword to the actual function call which I think is more idiomatic